### ISSUE TITLE
Fail to work properly after configuring the Mesh using the console

### ISSUE THREAD

**Issue by iamhopaul123** [subfuzion/enable-appmesh#5]
After spinning up the basic construct using the CDK, it works just as expected. I followed the [tutorial](https://medium.com/containers-on-aws/enable-app-mesh-for-ecs-fargate-apps-with-the-aws-console-2bdc7c1b37dd) trying to configure the app mesh. However, it cannot work properly and when I run "curl $demo/color", it will show like `{"color":"no healthy upstream", "stats": {"no healthy upstream":1}}`

**Comment by subfuzion** [comment]
Yes, this is normal, but I should document better that it will say this for a short (indeterminate) period while Cloud Map updates take effect after starting the updated (mesh-enabled) services. At some point (which you can view in the Cloud Map console), the new tasks will all report healthy.

In the current Go code for the Color App demo, however, once it sees and records this as part of the history, it will never disappear unless you call the `/color/clear` route to clear it. After that, the responses should look normal again.

**Comment by subfuzion** [comment]
I think I will do a screen recording and attach a link so that it's clearer exactly what people should see. I don't think the documentation adequately conveys that as much as I would like.

**Comment by iamhopaul123** [comment]
>Yes, this is normal, but I should document better that it will say this for a short (indeterminate) period while Cloud Map updates take effect after starting the updated (mesh-enabled) services. At some point (which you can view in the Cloud Map console), the new tasks will all report healthy.

Yeah I think it takes a while for the previous gateway task get deregistered from the target group and then get stopped. However, even if all tasks are healthy, it still show like this `{"color":"no healthy upstream", "stats": {"no healthy upstream":1}}`. Not sure if it is because of ECS or App mesh setting.
